osquery/tables/system/linux/processes.cpp

```diff
@@ -287,6 +287,46 @@ SimpleProcStat::SimpleProcStat(const std::string& pid) {
   }
 }
 
+/**
+ * Output from string parsing /proc/<pid>/io.
+ */
+struct SimpleProcIo : private boost::noncopyable {
+public:
+    std::string read_bytes;
+    std::string write_bytes;
+    std::string cancelled_write_bytes;
+    
+    /// For errors processing proc data.
+    Status status;
+    
+    explicit SimpleProcIo(const std::string& pid);
+};
+
+SimpleProcIo::SimpleProcIo(const std::string& pid) {
+    std::string content;
+    if (!readFile(getProcAttr("io", pid), content).ok()) {
+        status = Status(1, "Cannot read /proc/<pid>/io");
```

Comment:
If you know the PID, why not write it directly into the status message. Then the `VLOG` becomes simpler.